examples/basic/main.tf

```diff
@@ -6,10 +6,57 @@ resource "random_id" "this" {
   byte_length = 8
 }
 
-module "foobar" {
-  # source = "github.com/equinor/terraform-azurerm-foobar?ref=v0.0.0"
-  source = "../.."
+resource "random_password" "this" {
+  length           = 72
+  lower            = true
+  upper            = true
+  numeric          = true
+  special          = true
+  override_special = "!@#$%&_-+="
+  min_lower        = 1
+  min_upper        = 1
+  min_numeric      = 1
+  min_special      = 1
+}
```

Comment:
Could this be moved into the module itself? Similar to Azure SQL module: https://github.com/equinor/terraform-azurerm-sql/blob/2cc09d452f83c0f1c1738082b613c34e71d30691/main.tf#L1-L11

And do you have a link to official documentation to clarify that this is the official spec for a MySQL password? We could add it as a comment for future reference.